contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -288,17 +303,17 @@ contract LidoOracle is ILidoOracle, IsContract, AragonApp {
             uint256 lastReportableEpochId
         )
     {
+        uint256 earliestReportableEpochId = (
+            EARLIEST_REPORTABLE_EPOCH_ID_VALUE_POSITION.getStorageUint256()
+        );
         return (earliestReportableEpochId, getCurrentEpochId());
     }
 
-    /**
-     * @dev Pushed the current data point if quorum is reached
-     */
-    function _tryPush(uint256 _epochId) internal {
+    function _isQuorum(uint256 _epochId) internal returns (bool, Report) {
```

Comment:
This function should be marked as `view`.

Reply:
Also, since it returns the report I'd rename it to smth like `_getQuorumReport`. Naming it `_isQuorum` misleadingly implies that it returns a bool.

Reply:
One more thing: I'd explicitly specify the `memory` storage location for the report, i.e. `returns (bool, Report memory)`.

Reply:
Fixed and added description